web-demo/doodle_game/backend/build.gradle

```diff
@@ -10,10 +10,10 @@ repositories {
 }
 
 dependencies {
-    implementation platform("ai.djl:bom:0.10.0")
+    implementation platform("ai.djl:bom:0.20.0")
     implementation "ai.djl:api"
     implementation "ai.djl.pytorch:pytorch-engine"
-    implementation "ai.djl.pytorch:pytorch-native-auto"
+    implementation "ai.djl.pytorch:pytorch-native-cpu"
```

Comment:
remove

Reply:
This is pytorch-engine dependency, should we keep the native?